[
   {
      "author_association" : "CONTRIBUTOR",
      "body" : "<!--e57a25ab6845829454e8d69fc972939a-->\n\nThe following sections might be updated with supplementary metadata relevant to reviewers and maintainers.\n\n<!--006a51241073e994b41acfe9ec718e94-->\n### Code Coverage\nFor detailed information about the code coverage, see the [test coverage report](https://corecheck.dev/bitcoin/bitcoin/pulls/28486).\n<!--021abf342d371248e50ceaed478a90ca-->\n### Reviews\nSee [the guideline](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#code-review) for information on the review process.\nA summary of reviews will appear here.\n",
      "created_at" : "2023-09-14T22:08:03Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1720218312",
      "id" : 1720218312,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585miHLI",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1720218312/reactions"
      },
      "updated_at" : "2023-10-16T16:14:43Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1720218312",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/39886733?v=4",
         "events_url" : "https://api.github.com/users/DrahtBot/events{/privacy}",
         "followers_url" : "https://api.github.com/users/DrahtBot/followers",
         "following_url" : "https://api.github.com/users/DrahtBot/following{/other_user}",
         "gists_url" : "https://api.github.com/users/DrahtBot/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/DrahtBot",
         "id" : 39886733,
         "login" : "DrahtBot",
         "node_id" : "MDQ6VXNlcjM5ODg2NzMz",
         "organizations_url" : "https://api.github.com/users/DrahtBot/orgs",
         "received_events_url" : "https://api.github.com/users/DrahtBot/received_events",
         "repos_url" : "https://api.github.com/users/DrahtBot/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/DrahtBot/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/DrahtBot/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/DrahtBot"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "> For some reasons (I failed to figure them out, unfortunately) the code on the master branch works.\r\n\r\n@sipsorcery Maybe you have any insights about that?",
      "created_at" : "2023-09-14T22:09:02Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1720219268",
      "id" : 1720219268,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585miHaE",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1720219268/reactions"
      },
      "updated_at" : "2023-09-14T22:09:02Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1720219268",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "I'm having trouble understanding the problem here. Could you please explain what the resulting error is?\r\n\r\nI can understand possibly needing to move `SetupNetworking()` to be more correct. But I don't understand what CMake has anything to do with this. IMO we need to figure out _why_ before moving anything around.",
      "created_at" : "2023-09-19T10:26:23Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1725237288",
      "id" : 1725237288,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585m1Qgo",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725237288/reactions"
      },
      "updated_at" : "2023-09-19T10:26:23Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725237288",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/417043?v=4",
         "events_url" : "https://api.github.com/users/theuni/events{/privacy}",
         "followers_url" : "https://api.github.com/users/theuni/followers",
         "following_url" : "https://api.github.com/users/theuni/following{/other_user}",
         "gists_url" : "https://api.github.com/users/theuni/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/theuni",
         "id" : 417043,
         "login" : "theuni",
         "node_id" : "MDQ6VXNlcjQxNzA0Mw==",
         "organizations_url" : "https://api.github.com/users/theuni/orgs",
         "received_events_url" : "https://api.github.com/users/theuni/received_events",
         "repos_url" : "https://api.github.com/users/theuni/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/theuni/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/theuni/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/theuni"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "Looking at your commit that purposefully introduces a bug, it removes a bunch of bench files. I'm guessing one of those happens to call `WSAStartup` somehow?",
      "created_at" : "2023-09-19T10:43:54Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1725260724",
      "id" : 1725260724,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585m1WO0",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725260724/reactions"
      },
      "updated_at" : "2023-09-19T10:43:54Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725260724",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/417043?v=4",
         "events_url" : "https://api.github.com/users/theuni/events{/privacy}",
         "followers_url" : "https://api.github.com/users/theuni/followers",
         "following_url" : "https://api.github.com/users/theuni/following{/other_user}",
         "gists_url" : "https://api.github.com/users/theuni/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/theuni",
         "id" : 417043,
         "login" : "theuni",
         "node_id" : "MDQ6VXNlcjQxNzA0Mw==",
         "organizations_url" : "https://api.github.com/users/theuni/orgs",
         "received_events_url" : "https://api.github.com/users/theuni/received_events",
         "repos_url" : "https://api.github.com/users/theuni/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/theuni/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/theuni/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/theuni"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "> Looking at your commit that purposefully introduces a bug, it removes a bunch of bench files. I'm guessing one of those happens to call `WSAStartup` somehow?\r\n\r\n~That was my initial guess. But it is not the case. I mean, there are no _explicit_ `WSAStartup` invocations.~\r\n\r\nYes, in the `MakeNoLogFileContext` template function where the compiler decided to instantiate the `BasicTestingSetup` struct.",
      "created_at" : "2023-09-19T10:46:40Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1725264367",
      "id" : 1725264367,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585m1XHv",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725264367/reactions"
      },
      "updated_at" : "2023-10-16T16:14:41Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725264367",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "The error in that github action is:\r\n```\r\nError: Bad optional access\r\nError: Process completed with exit code 1.\r\n```\r\n\r\nWhich implies to me that something is throwing `std::bad_optional_access`. That seems to be something different from what you're talking about?",
      "created_at" : "2023-09-19T13:53:15Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1725627180",
      "id" : 1725627180,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585m2vss",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725627180/reactions"
      },
      "updated_at" : "2023-09-19T13:53:15Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725627180",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/417043?v=4",
         "events_url" : "https://api.github.com/users/theuni/events{/privacy}",
         "followers_url" : "https://api.github.com/users/theuni/followers",
         "following_url" : "https://api.github.com/users/theuni/following{/other_user}",
         "gists_url" : "https://api.github.com/users/theuni/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/theuni",
         "id" : 417043,
         "login" : "theuni",
         "node_id" : "MDQ6VXNlcjQxNzA0Mw==",
         "organizations_url" : "https://api.github.com/users/theuni/orgs",
         "received_events_url" : "https://api.github.com/users/theuni/received_events",
         "repos_url" : "https://api.github.com/users/theuni/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/theuni/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/theuni/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/theuni"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "> The error in that github action is:\r\n> \r\n> ```\r\n> Error: Bad optional access\r\n> Error: Process completed with exit code 1.\r\n> ```\r\n> \r\n> Which implies to me that something is throwing `std::bad_optional_access`. That seems to be something different from what you're talking about?\r\n\r\nThe call chain from here https://github.com/bitcoin/bitcoin/blob/459272d639b9547f68000d2b9a5a0d991d477de5/src/bench/addrman.cpp#L124 is as follows:\r\n- `std::optional<CNetAddr> LookupHost(const std::string&, bool, DNSLookupFn)`\r\n- `std::vector<CNetAddr> LookupHost(const std::string&, unsigned int, bool, DNSLookupFn)`\r\n- `std::vector<CNetAddr> LookupIntern(const std::string&, unsigned int, bool, DNSLookupFn)`\r\n- `std::vector<CNetAddr> WrappedGetAddrInfo(const std::string&, bool)`\r\n- [`getaddrinfo`](https://learn.microsoft.com/en-us/windows/win32/api/ws2tcpip/nf-ws2tcpip-getaddrinfo)\r\n\r\nIf the latter fails, for example, due to uninitialized Winsock, the `WrappedGetAddrInfo` returns empty vector, and the first `LookupHost` does `std::nullopt`, which in turn causes `Error: Bad optional access`.",
      "created_at" : "2023-09-19T14:14:33Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1725692743",
      "id" : 1725692743,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585m2_tH",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725692743/reactions"
      },
      "updated_at" : "2023-09-19T14:14:33Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1725692743",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "In this [branch](https://github.com/hebasto/bitcoin/tree/230923-demo-WSANOTINITIALISED-error), I've minimized the problem code to a [single line of code](https://github.com/hebasto/bitcoin/blob/58b08c97beef8041985edbab157202164714304b/src/bench/bench_bitcoin.cpp#L28-L29), which is:\r\n```c++\r\n    // Comment the next line out to trigger the WSANOTINITIALISED=10093 error.\r\n    const auto test_setup = MakeNoLogFileContext<const TestingSetup>();\r\n```",
      "created_at" : "2023-09-23T20:51:04Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1732406740",
      "id" : 1732406740,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585nQm3U",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1732406740/reactions"
      },
      "updated_at" : "2023-09-23T20:51:04Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1732406740",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "I still don't understand why our CI isn't failing, in the way you claim it should be, if our binaries are meant to be broken.\r\n\r\nNothing should be done here, until the root cause of the issue is actually understood. As this either points to some brokenness in master (build system/CI), or your CMake/MSVC branch producing binaries differently to master.",
      "created_at" : "2023-09-24T16:42:03Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1732616749",
      "id" : 1732616749,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585nRaIt",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1732616749/reactions"
      },
      "updated_at" : "2023-09-24T16:42:03Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1732616749",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/863730?v=4",
         "events_url" : "https://api.github.com/users/fanquake/events{/privacy}",
         "followers_url" : "https://api.github.com/users/fanquake/followers",
         "following_url" : "https://api.github.com/users/fanquake/following{/other_user}",
         "gists_url" : "https://api.github.com/users/fanquake/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/fanquake",
         "id" : 863730,
         "login" : "fanquake",
         "node_id" : "MDQ6VXNlcjg2MzczMA==",
         "organizations_url" : "https://api.github.com/users/fanquake/orgs",
         "received_events_url" : "https://api.github.com/users/fanquake/received_events",
         "repos_url" : "https://api.github.com/users/fanquake/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/fanquake/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/fanquake"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "> I still don't understand why our CI isn't failing, in the way you claim it should be, if our binaries are meant to be broken.\r\n\r\nOur code is incorrect according to Microsoft's docs because the `SetupNetworking` (with `WSAStartup` in it) is never called in `bench_bitcoin.exe` binary.",
      "created_at" : "2023-09-24T17:06:31Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1732621625",
      "id" : 1732621625,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585nRbU5",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1732621625/reactions"
      },
      "updated_at" : "2023-09-24T17:06:31Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1732621625",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   },
   {
      "author_association" : "MEMBER",
      "body" : "> I still don't understand why our CI isn't failing, in the way you claim it should be, if our binaries are meant to be broken.\r\n\r\nApparently, the underlying problem is not MSVC or CMake specific. I've opened a dedicated issue with steps to reproduce the bug.\r\n\r\nWe might adjust our CI to catch the problem easily.\r\n\r\n---\r\n\r\nThe PR description has been updated.",
      "created_at" : "2023-11-25T20:56:00Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/pull/28486#issuecomment-1826418116",
      "id" : 1826418116,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28486",
      "node_id" : "IC_kwDOABII585s3O3E",
      "performed_via_github_app" : null,
      "reactions" : {
         "+1" : 0,
         "-1" : 0,
         "confused" : 0,
         "eyes" : 0,
         "heart" : 0,
         "hooray" : 0,
         "laugh" : 0,
         "rocket" : 0,
         "total_count" : 0,
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1826418116/reactions"
      },
      "updated_at" : "2023-11-25T20:56:00Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/1826418116",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/32963518?v=4",
         "events_url" : "https://api.github.com/users/hebasto/events{/privacy}",
         "followers_url" : "https://api.github.com/users/hebasto/followers",
         "following_url" : "https://api.github.com/users/hebasto/following{/other_user}",
         "gists_url" : "https://api.github.com/users/hebasto/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/hebasto",
         "id" : 32963518,
         "login" : "hebasto",
         "node_id" : "MDQ6VXNlcjMyOTYzNTE4",
         "organizations_url" : "https://api.github.com/users/hebasto/orgs",
         "received_events_url" : "https://api.github.com/users/hebasto/received_events",
         "repos_url" : "https://api.github.com/users/hebasto/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/hebasto/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/hebasto/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/hebasto"
      }
   }
]
